### PR TITLE
Extract Pane component

### PR DESCRIPTION
**Pull request by dgalarza**
A pane requires a background and a boxShadow. Both have defaults provided.

* Card composes Pane with a borderRadius set.
* Add white as a background color
* Popover content doens't need to specify a borderRadius since card already has one.

image

### CODE REVIEW

File: src/Popover/Popover.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ import Card from '../Card';
 
 const PopoverContent = styled(Card)`
   background: ${props => props.theme.colors.white};
-  border-radius: ${props => props.theme.radii.large};
```
Comment (acostanzo) [inline src/Popover/Popover.js]: Just a heads up, I moved PopoverContent into its own file in my datepicker PR.

Comment (acostanzo) [reply src/Popover/Popover.js]: Maybe you wanted them next to each other since the differences is so small?

**Comment by acostanzo** [comment]
> LGTM. Just curious, why put the cards and panes in the same story?

Maybe you wanted them next to each other since the difference is so small?

File: src/Popover/Popover.js (hunk 1 above)
Comment (dgalarza) [reply src/Popover/Popover.js]: It could probably be done in separate stories, I just ended up doing so to match the figma design which has them together.

**Comment by dgalarza** [comment]
@acostanzo I went ahead and extracted panes from the cards stories so that it has it's own story.

File: src/Popover/Popover.js [hunk 2]
```diff
@@ -10,7 +10,6 @@ import Card from '../Card';
...
   box-shadow: 0 4px 8px
```
Comment (mayravherman) [inline src/Popover/Popover.js]: Popovers are cards with elevation2. If we add that default we can remove boxshadow from here @dgalarza

Comment (dgalarza) [reply src/Popover/Popover.js]: Going to tackle this as a separate PR. Created a story here - https://www.pivotaltracker.com/story/show/164466979

**Pull request merged**